packages/rancher-gatekeeper/overlay/app-readme.md

```diff
@@ -0,0 +1,14 @@
+# Rancher OPA Gatekeeper
+
+This chart is based off of the upstream [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper/tree/master/charts/gatekeeper) chart.
+
+For more information on how to use the feature, refer to our [docs](https://rancher.com/docs/rancher/v2.x/en/opa-gatekeper/).
```

Comment:
This Docs page still mentions OPA as experimental feature of Rancher - Should we be changing the docs to mention the GA status since Rancher v2.5?
<img width="962" alt="Screen Shot 2020-11-02 at 4 33 08 PM" src="https://user-images.githubusercontent.com/1470602/97934011-41d0ce00-1d29-11eb-91f5-ce6d9eea2b5c.png">

Reply:
It's still experimental in 2.5 @prachidamle